src/backend/tests/Fusion.Testing.Mocks.ProfileService/Api/PersonsController.cs

```diff
@@ -19,6 +19,7 @@ namespace Fusion.Testing.Mocks.ProfileService.Api
     public class PersonsController : ControllerBase
     {
         [MapToApiVersion("3.0")]
```

Comment:
in this test, do we need the old api-version ?

Reply:
yes, but new version is not required